### ISSUE TITLE
Error script not loading

### ISSUE THREAD

**Issue by Stargate-project** [umami-software/umami#1511]
the script is not loading on brave browsers, anyway around this?

```
Failed to load resource: net::ERR_BLOCKED_BY_CLIENT

```

**Comment by Stargate-project** [comment]
after disabling the shields I get more errors...

```
Access to XMLHttpRequest at 'https://analytics.com/api/collect' from origin 'https://tiempo.com' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
umami.js:1          


POST https://analytics.com/api/collect net::ERR_FAILED

```

**Comment by Stargate-project** [comment]
is it possible to add image tracking?

**Comment by JasonRodriguez1474** [comment]
For this error, "Failed to load resource: net::ERR_BLOCKED_BY_CLIENT"
Please see the following closed issue, #1498 
Ultimately, it's likely being caused by an adblocker from your browser

**Comment by Stargate-project** [comment]
I added this to my env file but nothing changes in my umami admin

```
TRACKER_SCRIPT_NAME = asflskdjf

```

**Comment by Stargate-project** [comment]
after changing the TRACKER_SCRIPT_NAME I get a 404 error

```
Failed to load resource: the server responded with a status of 403 ()

```

**Comment by jonaskuske** [comment]
@Stargate-project You need to change `COLLECT_API_ENDPOINT` in addition to `TRACKER_SCRIPT_NAME` as that's blocked as well

**Comment by franciscao633** [comment]
Closing out old issues. If you continue to experience errors, please open a new issue.
